custom_components/homematicip_local/entity_helpers.py

```diff
@@ -174,7 +174,7 @@
     "GAS_FLOW": HmSensorEntityDescription(
         key="GAS_FLOW",
         native_unit_of_measurement=UnitOfVolume.CUBIC_METERS,
```

Comment:
I think this is still wrong. needs to be `m³/h`

> using native unit of measurement 'm³' which is not a valid unit for the device class ('volume_flow_rate') it is using; expected one of ['L/min', 'gal/min', 'm³/h', 'ft³/min']

Reply:
Maybe, I don't care until someone adds a device description.

> Please attach the device definition as described [here](https://github.com/danielperna84/custom_homematic#homematicip_localexport_device_definition).

Reply:
Hello, these device description files are uploaded to the issue [#1453](https://github.com/danielperna84/hahomematic/issues/1453) and referenced in the "Additional information" section.

Reply:
Thanks